### DISCUSSION TITLE
有方法能检验程序能否正常抢票吗

### DISCUSSION THREAD

**Discussion by EatablePaimon** [mikumifa/biliTickerBuy#250] [提问专区]
![屏幕截图 2024-06-29 171822](https://github.com/mikumifa/biliTickerBuy/assets/129951951/940661c8-6bfa-4d09-b216-70e37b093e13)
shi'l 试了下上海·燃动!!高梨康治SUMMER LIVE TOUR IN CHINA 2024 这个的抢票 但是点击开始抢票没有什么反应 有什么办法检验能正确抢票嘛

**Comment by mikumifa** [comment]
看一下控制台
